addons/mail/static/src/core/common/thread_service.js

```diff
@@ -867,10 +869,22 @@ export class ThreadService {
         }
         if (thread?.model === "discuss.channel") {
             if (persona.type === "partner") {
-                return url(`/discuss/channel/${thread.id}/partner/${persona.id}/avatar_128`, urlParams);
+                return url("/web/image", {
```

Comment:
This should become "persona.avatarUrl", not repeating code and especially not in thread.